nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -196,6 +196,54 @@ class OVSplitMetatype(OVOpMetatype):
     hw_config_names = [HWConfigOpName.SPLIT]
 
 
+@OV_OPERATION_METATYPES.register()
+class OVVariadicSplitMetatype(OVOpMetatype):
+    name = 'VariadicSplitOp'
+    op_names = ['VariadicSplit']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVShuffleChannelsMetatype(OVOpMetatype):
+    name = 'ShuffleChannelsOp'
+    op_names = ['ShuffleChannels']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVBroadcastMetatype(OVOpMetatype):
+    name = 'BroadcastOp'
+    op_names = ['Broadcast']
+
+
+@OV_OPERATION_METATYPES.register()
+class OVConvertLikeMetatype(OVOpMetatype):
+    name = 'ConvertLikeOp'
+    op_names = ['ConvertLike']
```

Comment:
Is it the same that the `Convert` operation?

Reply:
I think yes according to the [documentation](https://docs.openvino.ai/latest/openvino_docs_ops_type_ConvertLike_1.html#doxid-openvino-docs-ops-type-convert-like-1).